package/piwebcam/video_formats.txt

```diff
@@ -0,0 +1,16 @@
+# Default Show-me webcam supported resolutions
+#
+# format: video format(mjpeg|raw|mp4) x-dimension y-dimension
```

Comment:
mp4 is not yet supported unfortunately, see https://github.com/peterbay/uvc-gadget/issues/8

Currently we only have `mjpeg` and `uncompressed`, see: https://github.com/raspberrypi/linux/blob/rpi-5.4.y/drivers/usb/gadget/function/uvc_configfs.c#L767

Reply:
I corrected that in fb0418a